lib/utils.js

```diff
@@ -97,10 +97,11 @@ export function getEmbeddedScope(
   return _.find(scopes, s => s.indexOf("source.embedded.") === 0);
 }
 
-export function getEditorDirectory(editor: atom$TextEditor) {
-  const editorPath = editor.getPath();
-  return editorPath ? path.dirname(editorPath) : os.homedir();
+// I feel like this should work
+export function getEditorDirectory(editor: ?atom$TextEditor) {
+  return editor ? path.dirname(editor.getPath()) : os.homedir();
```

Comment:
I think we should handle the case `editor == null` higher up. I'm not sure if this works. `editor.getPath()` still can return undefined.